src/foundry/client/apps/forms/journal-page-sheet.d.ts

```diff
@@ -0,0 +1,373 @@
+import type { EditorView } from "prosemirror-view";
+import type Showdown from "showdown";
+import type { Editor } from "tinymce";
+import type { CONST } from "../../../common/module.mjs.js";
+
+declare global {
+  /**
+   * The Application responsible for displaying and editing a single {@link JournalEntryPage} document.
+   *
+   * @typeParam ConcreteOptions - The type of the options object
+   */
+  class JournalPageSheet<
+    ConcreteOptions extends JournalPageSheet.Options = JournalPageSheet.Options
+  > extends DocumentSheet<ConcreteOptions, JournalEntryPage> {
+    /**
+     * @defaultValue
+     * ```typescript
+     * foundry.utils.mergeObject(super.defaultOptions, {
+     *   classes: ["sheet", "journal-sheet", "journal-entry-page"],
+     *   width: 600,
+     *   height: 680,
+     *   resizable: true,
+     *   closeOnSubmit: false,
+     *   submitOnClose: true,
+     *   viewPermission: CONST.DOCUMENT_OWNERSHIP_LEVELS.OBSERVER
+     * });
+     * ```
+     */
+    static get defaultOptions(): JournalPageSheet.Options;
+
+    get template(): string;
+
+    get title(): string;
+
+    activateEditor(
+      name: string,
+      options?: TextEditor.Options | undefined,
+      initialContent?: string | undefined
+    ): Promise<Editor | EditorView>;
+
+    override getData(options?: Partial<ConcreteOptions> | undefined): Promise<JournalPageSheet.Data<ConcreteOptions>>;
+
+    protected _renderInner(data: object): Promise<JQuery<HTMLElement>>;
+  }
+
+  namespace JournalPageSheet {
+    // eslint-disable-next-line @typescript-eslint/no-empty-interface
+    interface Options extends DocumentSheetOptions {}
+    interface Data<T extends JournalPageSheet.Options | undefined = JournalPageSheet.Options | undefined> {
```

Comment:
Adding a separate entry for an Options interface here is actually superfluous. We just used the parent sheet's options in the past. Also as long as you are in the namespace `JournalPageSheet`, you should be able to reference it as just `Options`.